pyproject.toml

```diff
@@ -70,22 +70,22 @@ python = ">=3.6,<3.9"
 boto3 = "^1.12"
 requests = "^2.23"
 matplotlib = ">=3.1,<3.4"
-attrs = ">=19.3,<20.3"
+attrs = ">=19.3,<20.4"
```

Comment:
could we also bump the range for `black` to the `^20.0`? It's currently a bit annoying to do development for the Rasa SDK as the range are different and I've to uninstall the black formatter each time 😁